iib/web/models.py

```diff
@@ -698,6 +704,15 @@ def _from_json(
                     ' the "overwrite_from_index_token" parameter is used'
                 )
 
+        distribution_scope = request_kwargs.pop('distribution_scope', None)
+        if distribution_scope:
+            distribution_scope = distribution_scope.lower()
+            if distribution_scope not in ['prod', 'stage', 'dev']:
+                raise ValidationError(
+                    'The "distribution_scope" parameter must be one of "dev", "stage", or "prod"'
```

Comment:
Optional: This sounds a little more intuitive when talking about JSON.
```suggestion
                    'The "distribution_scope" value must be one of "dev", "stage", or "prod"'
```